kube/services/revproxy/00nginx-config.yaml

```diff
@@ -139,7 +139,15 @@ data:
       	##
         gzip on;
         gzip_disable "msie6";
-        gzip_types application/javascript;
+        gzip_proxied any;
+        gzip_types
+          text/css
+          text/javascript
+          text/xml
+          text/plain
+          application/javascript
+          application/x-javascript
+          application/json;
```

Comment:
I think we decided not to gzip json due to the BREACH attack:
   https://en.wikipedia.org/wiki/BREACH

Reply:
I don't believe we actually reflect user data anywhere so we shouldn't be vulnerable to this?

Reply:
Actually thats not true, we reflect names in graphql queries so you could potentially do it there. Disabling for the graphql endpoint